test/unit/compute_resources/ovirt_test.rb

```diff
@@ -24,4 +24,29 @@ class OvirtTest < ActiveSupport::TestCase
       assert_find_by_uuid_raises(ActiveRecord::RecordNotFound, cr)
     end
   end
+
+  describe "associating operating system" do
+    setup do
+      @operating_systems_xml = Nokogiri::XML(File.read('test/fixtures/ovirt_operating_systems.xml'))
```

Comment:
I would do that if `rbovirt` would be build in a way that the model objects don't take the xmls in arguments https://github.com/abenari/rbovirt/blob/master/lib/ovirt/storage_domain.rb#L5. I don't want to introduce other approach to rbovirt just for operating systems. Doing this parsing takes `0.000337103 s` on my laptop and I bet jenkins is not much worse on this.